caribou/deployment_solver/deployment_metrics_calculator/models/workflow_instance.py

```diff
@@ -215,9 +215,6 @@ def add_node(self, instance_index: int) -> bool:
                 current_node.cumulative_runtimes,
                 current_node.execution_time,
                 data_transfer_during_execution,
-                # ) = self._input_manager.get_node_runtimes_and_data_transfer(
```

Comment:
Delete?

Reply:
This was leftover code that was not deleted previously.